msgraph-mail/ruby/graphrubyv4/api_client.rb

```diff
@@ -1,10 +1,10 @@
-require './graphrubyv4'
+require './graph_c'
```

Comment:
This file seems not to have been updated with `graphrubyv4`.

Is there a way to run a compiler or the equivalent of that in Ruby here to check the type integrity of the generated files?